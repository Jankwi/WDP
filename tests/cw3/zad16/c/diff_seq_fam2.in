10 1 6 -3 2 10 12 3 5 24 0