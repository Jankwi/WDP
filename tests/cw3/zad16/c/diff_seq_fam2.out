5.000000 -9.000000 5.000000 8.000000 2.000000 -9.000000 2.000000 19.000000 -24.000000